### PR TITLE
upgrade http link to https

### PR DESCRIPTION
**Pull request by eatjeff**

### CODE REVIEW

File: content/id/community/static/cncf-code-of-conduct.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Contoh perilaku kasar, melecehkan, atau tidak dapat diterima di Kubernetes dapat
 
 Kode Etik ini diadaptasi dari Covenant Contributor
 <http://contributor-covenant.org>, versi 1.2.0, tersedia di
```
Comment (jihoon-seo) [inline content/id/community/static/cncf-code-of-conduct.md:26]:
```suggestion
<https://contributor-covenant.org>, versi 1.2.0, tersedia di
```

**Review by ariscahyadi — COMMENTED** [review]
Thanks @eatjeff and @jihoon-seo to add this statement.

**Pull request merged**